pkg/kf/commands/install/gke/gke.go

```diff
@@ -527,6 +530,21 @@ func gkeClusterConfig(
 		}
 	}
 
+	// Release Channel
+	_, wantSpecificVersion := os.LookupEnv(GKEVersionEnvVar)
```

Comment:
Given everywhere else we're using flags, could we switch it to a flag instead or is that outside the scope of this PR?

Reply:
I think it's outside of scope here and we should talk about if we need this and if we still do how we should discourage/hide it from users.